ethstorage/p2p/protocol/syncserver.go

```diff
@@ -311,3 +320,7 @@ func (srv *SyncServer) HandleRequestShardList(ctx context.Context, log log.Logge
 	}
 	log.Debug("Write response done for HandleRequestShardList")
 }
+
+func (srv *SyncServer) ProvidedBlobs() map[uint64]uint64 {
+	return srv.providedBlobs
```

Comment:
This is a different handling way with the sync client, and also it is different with the mining status